includes/wc-stock-functions.php

```diff
@@ -0,0 +1,157 @@
+<?php
+/**
+ * WooCommerce Stock Functions
+ *
+ * Functions used to manage product stock levels.
+ *
+ * @author 		WooThemes
+ * @category 	Core
+ * @package 	WooCommerce/Functions
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Update a product's stock amount.
+ *
+ * Uses queries rather than update_post_meta so we can do this in one query (to avoid stock issues).
+ *
+ * @since  2.7.0 this supports set, increase and decrease.
+ * @param  int|WC_Product $product
+ * @param  int|WP_Error $new_stock_level or an error if stock could not be set.
```

Comment:
Parameter is called $stock_quantity -- and it looks like its int or null, not int or WP_Error.